pkg/maps/fragmap/fragmap.go

```diff
@@ -69,6 +72,11 @@ func (v *FragmentValue) String() string {
 // map value.
 func (k FragmentKey) NewValue() bpf.MapValue { return &FragmentValue{} }
 
+// InitMapInfo updates the map info defaults for policy maps.
+func InitMapInfo(maxEntries int) {
+	MaxEntries = maxEntries
```

Comment:
I copied from what `PolicyMap` does, but no, no particular reason. I'll change it, although I'll keep `InitMapInfo()` for `PolicyMap`. This is because 1) it would require to change arg list for `Open()`, `OpenAndCreate()`, `Create()` in that package and pass the size every time we call one of them (`option` cannot be imported in `policymap`, circular import), and 2) `policymap.MaxEntries` is called from other packages so we want it to be initialised early to the value set by the user (and not wait until `newMap()` is called).